source/localizable/_code-of-conduct.it.md

```diff
@@ -0,0 +1,67 @@
+
+Comportamenti molesti includono: commenti scritti o verbali dannosi o pregiudizievoli in relazione al sesso, orientamento sessuale, razza, religione, disabilità fisiche e/o mentali; uso inappropriato di immagini di nudità e/o immagini a sfondo sessuale utilizzate in spazi pubblici (comprese le slides per le presentazioni); intimidazioni, il seguire fisicamente o stalking; molestie registrate sotto forma di fotografie e video; interruzione sostenuta di presentazioni o altri eventi; contatto fisico inappropriato, e sgradita attenzione sessuale.
+
+
+Conseguenze per i comportamenti inaccettabili
+-------------------------------------
+
+Qualsiasi comportamento ritenuto inaccettabile perpetuato da qualsiasi membro della community, compresi Sponsors e coloro nella posizione di potere decisionale, non sarà tollerato.
+Ci aspettiamo che chiunque venga ripreso nell' agire in maniera molesta, interrompa le suddette azioni immediatamente.
+
+Se un membro della community agisce in maniera non consona al Codice di Condotta, gli organizzatori della community possono reagire di conseguenza nel modo che più ritengono appropriato, incluso escludere temporaneamente o permanentemente il membro dalla community, senza preavviso alcuno (e senza rimborso in caso di evento a pagamento).
+
+
+Se noti o sei soggetto/a di un comportamento inaccettabile
+------------------------------------------------------
+
+Nel caso notassi o fossi soggetto/a a un comportamento inaccettabile, o avessi bisogno di chiarimenti, notifica uno/a degli/le organizzatori/ci il prima possibile.
+Puoi trovare una lista di organizzatori/ci da contattare per ognuno dei sostenitori di questo codice di condotta, alla fine di questa pagina.
+Inoltre, gli organizzatori/ci della community sono disponibili nell' aiutare i membri della community nel contattare le autorita' giudiziarie locali o rassicurare e ripristinare un livello di sicurezza adeguato per coloro bersaglio di comportamenti inaccettabili . In caso di eventi in persona, gli/le organizzatori/ci, nel caso colui/lei lo desideri, possono provvedere ad una scorta.
+
+
+Come gestire un reclamo
+---------------------
+
+Se pensi di essere stato falsamente o ingiustamente accusato di aver infranto il Codice di Condotta, dovresti immediatamente notificare uno degli organizzatori dell' evento con una descrizione concisa dell'accusa. Il tuo reclamo verrà gestito tenendo conto degli stessi termini delle nostre policies.
+
+
+Area di applicazione
+-----
+
+Ci aspettiamo che tutti i partecipanti alla community (contributori, stipendiati o meno; sponsors; e tutti gli ospiti) aderiscano a questo Codice di Condotta in tutti gli spazi appartenenti alla community, online, di persona ed in tutti i casi di comunicazione tra singoli pertinenti agli interessi della community.
```

Comment:
1. "tutti/e i/le"
2. "contributori/contributrici". 
3. avoid using ";" in this case. "contributori/contributrici stipendiati/e o meno, sponsor e tutti gli/le ospiti"